processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/RowBasedGrouperHelper.java

```diff
@@ -1819,6 +1896,12 @@ public BufferComparator getBufferComparator()
       {
         return bufferComparator;
       }
+
+      @Override
+      public Class<?> getClazz()
+      {
+        return Object.class;
```

Comment:
`String.class`?

Reply:
Similar explanation as to why I left it as Object.class to preserve existing behavior rather than being clearer and mentioning String.class. 
I doubt we'd run into regressions with the latter, but I was being overtly cautious.
wdyt?